### PR TITLE
Various small changes

### PR DESCRIPTION
**Pull request by abretaud**
Hi,
I'm currently testing genenotebook, it looks great!
Here's a PR with a few changes I've made to the code.

The main (visual) change is that I added external links and descriptions to GO terms and INTERPRO ids, it looks like that (I'm a beginner in react coding):

image

I've also changed the way default user accounts are created by using a more configurable `settings.json`.

As the bioconda package + docker image are not up-to-date, and to ease my life, I've also made a custom [docker image](https://github.com/abretaud/docker-gnb)

Is there any plans and/or a roadmap for a future release? I'm still doing more tests, and I have a few ideas, maybe we can discuss it if you're interested

Cheers

### CODE REVIEW

**Comment by abretaud** [comment]
Hi @holmrenser, have you seen this PR? I'd love to have some feedback if you have time :)

**Comment by holmrenser** [comment]
Hi @abretaud, Sorry I had missed it, thanks for the reminder! Looks like you have added some nice functionality, so I will go over the PR in detail for review.

Indeed the bioconda release is currently not up to date, my latest attempt at publishing to bioconda got frustrated by compiler issues. It's been a while, so I should check to see if things have improved and make a new release.

I have a few ideas for added functionality for a new release, so I am very interested in hearing about your ideas. Perhaps you can open a new issue with your ideas? One of the things I have been working is extracting all the visualisation components into a separate library: https://github.com/genenotebook/react-bio-viz. As you can see from that attempt, I would really like more flexible visualisation of trees and multiple sequence alignments. I am almost at a point where I can start swapping out genenotebook components with react-bio-viz components, so expect some activity there soon!

Again, thanks for the contribution!

File: imports/ui/genetable/columns/AttributeValue.jsx [hunk 1]
```diff
@@ -4,6 +4,47 @@ function isArray(x) {
   return Array.isArray(x) && x.length > 1;
 }
 
+function addExtUrl(value, setDescription) {
```
Comment (holmrenser) [inline imports/ui/genetable/columns/AttributeValue.jsx:7]: I don't think this has to be a separate function. By moving this logic to `DetailedSingleAttribute`, you can just have that component return the html anchor or plain value.

**Review by holmrenser — CHANGES_REQUESTED** [review]
I have a small suggestion on your implementation that would clean up the code a bit. Other than that I like the suggestions! The settings.json approach is nice, but could use some documentation. If you are not up for that I can make an attempt somewhere next week.

**Comment by abretaud** [comment]
Thanks for the feedback! I've just pushed the changes.
For settings.json, I can make a PR on https://github.com/genenotebook/genenotebook.github.io/ to document it

Nice to see https://github.com/genenotebook/react-bio-viz, that could be useful, even outside genenotebook!

For conda, maybe I can help at some point. In fact I made a small attempt to update it locally, but it miserably failed, and it was not top priority for me... maybe later

I'll try to sum up the ideas I have in mind in another issue :+1:

**Review by holmrenser — APPROVED** [review]
lgtm!

**Comment by abretaud** [comment]
Thanks for the merge!
How do you generate the pages in https://github.com/genenotebook/genenotebook.github.io/?

**Comment by holmrenser** [comment]
Thanks for the contribution!

The documentation is generated with [gatsby.js](https://www.gatsbyjs.com/), the source code is in the `src` branch of https://github.com/genenotebook/genenotebook.github.io/. You can run a development server by checking out the `src` branch and running `npm run dev`. If you submit a PR against the dev branch, I can then subsequently publish. Ideally this process will be done by a github action in the future.

Changing the docs site is currently not a well documented process. Guess I have to write some documentation for the documentation!

**Comment by abretaud** [comment]
Good, I'll look into it!

**Pull request merged**